xpsi/utilities/ProjectionTool.py

```diff
@@ -0,0 +1,648 @@
+            phiA.append(phi_c)
+            labels = [Mlab+SUPERlab, Mlab + CEDElab]
+
+        elif (HStag+REF['OR']) in DICT_VECTOR:
+            if not('C' in model):
+                print ("ERROR: C is not in model name, colatitude omission not present, omission ang. radius present")
+                raise IpyExit
+            thetaA.append(dictVp[HStag+REF['SC']])
+            zetaA.append(dictVp[HStag+REF['OR']])
+            phi_s = phiA[0]
+            phiA.append(phi_s)
+            labels = [Mlab+EMIlab, Mlab + OMIlab]
+
+        elif (HStag+REF['CR']) in DICT_VECTOR:
+            if not('C' in model):
+                print ("ERROR: C is not in model name, colatitude ceding not present, ceding ang. radius present")
+                raise IpyExit
+
+            thetaA.append(dictVp[HStag+REF['SC']])
+            zetaA.append(dictVp[HStag+REF['CR']])
+            phi_s = phiA[0]
+            phiA.append(phi_s)
+            TA.append(dictVp[HStag+REF['CT']])
+            labels = [Mlab+SUPERlab, Mlab + CEDElab]
+        labT_0 = " log(T/K) = %1.2f"%TA[0]
+        labels[0] = labels[0] +labT_0
+        if len(TA)>1:
+            labT_1 = " log(T/K) = %1.2f"%TA[1]
+            labels[1] =labels[1] +labT_1
+        return phiA,thetaA,zetaA,TA,labels
+
+
+
+    def SYMMETRIC(phiA,thetaA,labels):
+        phiA_s = np.zeros(len(phiA))
+        thetaA_s = np.zeros(len(thetaA))
+        labels_s = list(labels)
+
+        for i in range(len(phiA)):
+            phiA_s[i] = phiA[i]+0.5 if (phiA[i])<0.5 else phiA[i]-0.5
+            thetaA_s[i] = np.pi - thetaA[i]
+            labels_s[i] = labels_s[i].replace(Plab,Slab)
+
+        return phiA_s,thetaA_s,labels_s
+
+
+    phiA_p,thetaA_p,zetaA_p,TA_p,labels_p = fillVECTORS('p')
+
+
+    phiA_s = []
+    thetaA_s = []
+    zetaA_s = []
+    TA_s = []
+    labels_s =[]
+
+
+
+    CAall =[]
+    if '-S' in model:
+
+        phiA_s,thetaA_s,labels_s = SYMMETRIC(phiA_p,thetaA_p,labels_p)
+
+        zetaA_s = zetaA_p
+        TA_s = TA_p
+
+        if len(TA_p)==2:
+            iii = 0 if TA_p[0]>TA_p[1] else 6
+            CAall=[mycolors0[7-iii],mycolors0[iii],mycolors0[7-iii],mycolors0[iii]]
+        else:
+            if len(TA_p)==len(labels_p):
+                CAall=[mycolors0[6],mycolors0[6]]
+            else:
+                CAall=[mycolors0[6],'black',mycolors0[6],'black']
+
+    else:
+        phiA_s,thetaA_s,zetaA_s,TA_s,labels_s = fillVECTORS('s')
+        print ("phiA_s",phiA_s)
+        TAall = TA_p+TA_s
+        LABall = labels_p +labels_s
+        ind = [1,2,4,5] if len(TAall)>2 else [0,6]
+
+        argT = np.argsort(TAall)
+        j = 0
+        for i in range(len(LABall)):
+            if (OMIlab in LABall[i]):
+                coli = 'black'
+            else:
+                ind_i = ind[argT[j]]
+                coli = mycolors0[ind_i]
+                j = j+1
+            CAall.append(coli)
+            
+    if antipodal:
```

Comment:
I think here is at least an indentation error. There should be 4 more spaces before `if antipodal`.